src/modules/groups/groups.js

```diff
@@ -0,0 +1,92 @@
+const express = require('express');
+const axios = require('axios');
+
+const router = express.Router();
+
+const getAccessToken = async () => {
+  const { CLIENT_SECRET, CLIENT_ID, DEFAULT_MS_GRAPH_SCOPE, MS_LOGIN_HOST_NAME, TENANT_ID } =
+    process.env;
+  try {
+    const URL = `${MS_LOGIN_HOST_NAME}/${TENANT_ID}/oauth2/v2.0/token`;
+    const { data } = await axios.post(
+      URL,
+      {
+        grant_type: 'client_credentials',
+        client_secret: CLIENT_SECRET,
+        client_id: CLIENT_ID,
+        scope: DEFAULT_MS_GRAPH_SCOPE
+      },
+      {
+        headers: {
+          'Content-Type': 'multipart/form-data'
+        }
+      }
+    );
+    return { accessToken: data?.access_token };
```

Comment:
return `Bearer ${accessToken}`

Reply:
return accessToken ?  `Bearer ${accessToken}` : undefined

Reply:
THis is done.

Reply:
push the changes, i don't see the changes